src/sql/workbench/parts/notebook/cellViews/outputArea.css

```diff
@@ -10,4 +10,8 @@ output-area-component .notebook-output {
 	border-top-width: 0px;
 	user-select: text;
 	padding: 5px 20px 0px;
+}
+
+.output.actionselect {
```

Comment:
>.output.actionselect [](start = 0, length = 20)

You're adding the class "actionselect" in output.component.ts but here you're specifying the class as .output.actionselect - does that actually work as expected? Or is addClass doing something special?

Also please scope the classes as much as possible so the styles don't apply to things outside notebooks - something like this :

output-area-component .actionselect {

Reply:
👍 Please do some additional scoping as suggested here.